nix/test/basic/default.nix

```diff
@@ -0,0 +1,15 @@
+#
+# TODO: not sure if we need to import the sources
+#
+let
+  sources = import ./../../../nix/sources.nix;
```

Comment:
why leading './..' instead of just '../'

Reply:
that is nix-ism. All imports that refer to a file always start with "./" regardless of the path itself.